### PR TITLE
bug fix for #2 made default statement send warning instead of error

### PR DESCRIPTION
**Pull request by kapilkd13**
allowing default path to only throw warning instead of error. Additional changes are in Scheme-salad repo

### CODE REVIEW

**Comment by mr-c** [comment]
Jenkins, test this please

File: cwltool/schemas/v1.0/Process.yml [hunk 1]
```diff
@@ -562,7 +562,9 @@ $graph:
 
     - name: default
       type: Any?
-      jsonldPredicate: "cwl:default"
+      jsonldPredicate:
```
Comment (mr-c) [inline cwltool/schemas/v1.0/Process.yml]: Oh, the `cwltool/schemas` directory is a git subtree from https://github.com/common-workflow-language/common-workflow-language/ -- can you move your changes there?

Comment (kapilkd13) [inline cwltool/schemas/v1.0/Process.yml]: Hi @mr-c  Do you mean I should send a PR on https://github.com/common-workflow-language/common-workflow-language/ . Also should I make changes in v1.1.0-dev1 or v1.0
Thanks

Comment (mr-c) [inline cwltool/schemas/v1.0/Process.yml]: Yes, please send a new pull request to the other repo, with these changes applied to the `v1.0` and `v1.1.0-dev1` folders.

**Comment by tetron** [comment]
I think this is probably a good idea but it should be recorded as an v1.0 errata change.

**Comment by mr-c** [comment]
@tetron Thanks for the review -- I'll add the explanatory text when @kapilkd13 makes his new PR

**Comment by kapilkd13** [comment]
Hi @mr-c  I have made a PR to CWL repository as you suggested.

**Comment by mr-c** [comment]
Thanks! I'll close this PR and add some documentation to your PR.

**Pull request closed without merging**